src/test/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequestTest.java

```diff
@@ -42,4 +42,43 @@ void testFieldsAndAttachmentConstruction()
         assertEquals(CUSTOMER_EMAIL, chipsRequest.getCustomerEmail());
         assertEquals(REASON, chipsRequest.getReason());
     }
+
+    @Test
+    void testConstructionWithNullAttachment() {
+        ChipsRequest chipsRequest = new ChipsRequest(
+                OBJECTION_ID,
+                COMPANY_NUMBER,
+                null,
+                OBJECTION_ID,
+                CUSTOMER_EMAIL,
+                REASON,
+                DOWNLOAD_URL_PREFIX
+        );
+
+        // should not throw null pointer exception
+        assertNull(chipsRequest.getAttachments());
+    }
+
+    @Test
```

Comment:
I'm curious as to what gets output when the attachments are 'null'. I'm guessing 'null' features in the output string but worth checking/proving this with another unit-test I think.

Reply:
attachments=''. - I've now added a test to cover it